letsencrypt/README.md

```diff
@@ -1,11 +1,48 @@
 # The LetsEncrypt Role
-This document serves as a lightweight note about the role and its maintainance.
+This document serves as a lightweight note about the role and its maintenance.
 
+## How to Use this Role with Open edX
+This role, while bing generic enough to be used elsewhere, we're currently
```

Comment:
Typo, "bing" 
The wording is a little bit awkward, how about "This role is primarily used for Open edX deployments, but is generic enough to be used for other Debian based systems that use Nginx"?